### PR TITLE
Feature/26 paragraphs layout

### PR DESCRIPTION
**Pull request by finnlewis**
Moving to make use of drupal/layout_paragraphs for paragraphs with layout wrappers. 
Also patched core to fix nested paragraphs drag handles.

### CODE REVIEW

File: config/install/core.entity_form_display.node.localgov_campaigns_overview.default.yml [hunk 1]
```diff
@@ -95,6 +95,8 @@ third_party_settings:
         required_fields: false
       label: 'Child pages'
       region: content
+_core:
+  default_config_hash: gZQeaPTcYDi5mEr9WvBcVyS5ZbdPDoabz3ux7t_38PI
```
Comment (stephen-cox) [inline config/install/core.entity_form_display.node.localgov_campaigns_overview.default.yml:99]: These lines should be removed.

File: config/install/core.entity_form_display.node.localgov_campaigns_page.default.yml [hunk 2]
```diff
@@ -43,20 +43,8 @@ third_party_settings:
...
-      format_settings:
-        id: ''
-        classes: ''
-        formatter: closed
-        description: ''
-        required_fields: false
-      label: 'Page builder'
-      region: content
+_core:
+  default_config_hash: '-x6mhGIK5LIsPCXIjbJTqDRRp5BuDm5ya3XY-VYcjY0'
```
Comment (stephen-cox) [inline config/install/core.entity_form_display.node.localgov_campaigns_page.default.yml:47]: Also need removing.

File: config/install/core.entity_view_display.node.localgov_campaigns_overview.default.yml [hunk 3]
```diff
@@ -2,39 +2,43 @@ langcode: en
...
     - field.field.node.localgov_campaigns_overview.field_select_colorway
     - field.field.node.localgov_campaigns_overview.field_select_colourway_accent
     - field.field.node.localgov_campaigns_overview.localgov_campaigns_summary
     - node.type.localgov_campaigns_overview
   module:
-    - entity_reference_revisions
+    - layout_paragraphs
     - user
+_core:
+  default_config_hash: PXLbJkg8EK-TvjY6NZZFSFDjF6rfzqdI_WbDH62a2zc
```
Comment (stephen-cox) [inline config/install/core.entity_view_display.node.localgov_campaigns_overview.default.yml:19]: Need removing.

File: config/install/core.entity_view_display.node.localgov_campaigns_overview.teaser.yml [hunk 4]
```diff
@@ -3,17 +3,20 @@ status: true
...
-    - field.field.node.localgov_campaigns_overview.field_page_builder_advanced
+    - field.field.node.localgov_campaigns_overview.field_page_content
     - field.field.node.localgov_campaigns_overview.field_select_colorway
     - field.field.node.localgov_campaigns_overview.field_select_colourway_accent
     - field.field.node.localgov_campaigns_overview.localgov_campaigns_summary
     - node.type.localgov_campaigns_overview
   module:
     - user
+_core:
+  default_config_hash: 6KVHDA2Db5CkywsZbfDkfcxfAMRVLK_E3fXt9kxLXaw
```
Comment (stephen-cox) [inline config/install/core.entity_view_display.node.localgov_campaigns_overview.teaser.yml:19]: Remove

File: config/install/core.entity_view_display.node.localgov_campaigns_page.default.yml [hunk 5]
```diff
@@ -4,25 +4,28 @@ dependencies:
...
+    - field.field.node.localgov_campaigns_page.field_page_content
     - field.field.node.localgov_campaigns_page.field_topic_term
+    - field.field.node.localgov_campaigns_page.localgov_campaigns_summary
     - node.type.localgov_campaigns_page
   module:
-    - entity_reference_revisions
+    - layout_paragraphs
     - user
+_core:
+  default_config_hash: mJuMC5Xmmc90Nml546oi7N-OAyogeXtgz_cyilUFqhs
```
Comment (stephen-cox) [inline config/install/core.entity_view_display.node.localgov_campaigns_page.default.yml:15]: Remove

File: src/Plugin/Block/CampaignsNavigationBlock.php [hunk 6]
```diff
@@ -20,12 +20,14 @@
  *   admin_label = "Campaign navigation",
  * )
  */
-class CampaignsNavigationBlock extends CampaignsAbstractBlockBase {
+class CampaignsNavigationBlock extends CampaignsAbstractBlockBase
```
Comment (stephen-cox) [inline src/Plugin/Block/CampaignsNavigationBlock.php:23]: This class now contains quite a few coding standards errors
```
FILE: ...localgov_campaigns/src/Plugin/Block/CampaignsNavigationBlock.php
----------------------------------------------------------------------
FOUND 10 ERRORS AFFECTING 10 LINES
----------------------------------------------------------------------
  24 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
  30 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
  63 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
  77 | ERROR | [x] Expected newline after closing brace
 104 | ERROR | [x] Expected newline after closing brace
 120 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
 136 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
 155 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
 170 | ERROR | [x] Opening brace should be on the same line as the
     |       |     declaration
 185 | ERROR | [x] Expected 1 blank line after function; 0 found
```

**Comment by finnlewis** [comment]
Thanks @stephen-cox  - reviewing now

**Pull request merged**